api/controllers/admin.go

```diff
@@ -37,11 +38,11 @@ func (a *AdminController) Routes() []*fireball.Route {
 
 func (a *AdminController) GetConfig(c *fireball.Context) (fireball.Response, error) {
 	model := models.APIConfig{
-		Instance:       a.Config.Instance(),
-		VPCID:          a.Config.VPC(),
+		Instance:       a.Context.String(config.FlagInstance.GetName()),
```

Comment:
Is there any way we could simplify getting these values?

Reply:
Nothing that i can can think of that would require either:
* An interface, which would undo the main point of this pr
* Helper functions that would be more-or-less redundant, e.g. 
```go
foo := config.String(config.FlagInstance)
```

Reply:
I'm not a 100% sure here either. After an initial look:
* I will need more time to review this
* The trade-off for simplifying the config package is very apparent